### DISCUSSION TITLE
ARM64最终还是没有了……

### DISCUSSION THREAD

**Discussion by old-dog-pear** [Richasy/Bili.Copilot#75] [General]
哭死，请问开发者观止也是停止了么，昨天发现突然不能用了

**Comment by Richasy** [comment]
观止依赖于观止服务，这个早就停了。
ARM64的话，二维码生成有bug，先不支持了。Win11的话，也可以用x64版本的

**Reply by old-dog-pear** [reply]
好的，感谢开发者😊，rss追踪还会继续更新嘛，arm64版本的rss追踪依旧不能选择OneDrive，当然如果实在无法修复的话，也不影响使用😊，最后，再次感谢观止的陪伴，谢谢开发者的付出👍

**Comment by Richasy** [comment]
已经提供了ARM64支持，该话题关闭
